### ISSUE TITLE
[BUG] Custom :getter option for parameter results in invalid schema.

### ISSUE THREAD

**Issue by james-reading** [rswag/rswag#747]
## Describe the bug

Using the custom `:getter` option for parameters as described [here](https://github.com/rswag/rswag?tab=readme-ov-file#custom-getter-option-for-parameter), will introduce a `getter` key in the generated schema, and make the schema invalid.

In the spec :
```
parameter name: "labSlug", getter: :lab_slug, in: :path
```

In the generator schema, this results in:
```
"parameters": [
  {
    "name": "labSlug",
    "getter": "lab_slug",
    "in": "path",
    ...
```

Then when trying to use [orval](https://orval.dev/) to generate a client from the schema:
```
🍻 Start orval v6.27.1 - A swagger client generator for typescript
⚠️  SyntaxError: Swagger schema validation failed.
  #/paths/~[...]~1{labSlug}/parameters/0 must NOT have additional properties
```

## Expected behavior

I would expect the `getter` option to allow me to specific a differen variable for the path parameter. But when generating the schema with `rails rswag`, I do not expect this to be reflected in the generated schema file.

## Dependency versions
The version of are you using for:
* Rswag: 2.13.0
* RSpec: 3.13.0
* Rails: 7.1.3.2
* Ruby: 3.3.0

## Relates to which version of OAS (OpenAPI Specification)
- [ ] OAS2
- [x] OAS3
- [ ] OAS3.1

**Comment by brcebn** [comment]
This is correct. Also happening with OAS2 I would say. @romanblanco Do you think we could imagine to remove it from the generated swagger?

**Comment by azrosen92** [comment]
I think this issue can be closed. It looks like the PR that introduced the `:getter` argument handles this already https://github.com/rswag/rswag/pull/605/files#diff-a80181fc12bbecc8dfbac1c45647c2161358e36ad625eb50e800b113c427b5b5R216.
